frontend/src/views/FuelCodes/AddFuelCode/AddFuelCode.jsx

```diff
@@ -265,30 +216,17 @@ const AddFuelCodeBase = () => {
         </div>
         <BCGridEditor
           gridRef={gridRef}
-          columnDefs={fuelCodeColDefs(optionsData, errors)}
+          columnDefs={columnDefs}
           defaultColDef={defaultColDef}
           onGridReady={onGridReady}
           rowData={rowData}
-          gridOptions={gridOptions}
-          onCellEditingStopped={onCellEditingStopped}
           onCellValueChanged={onCellValueChanged}
-          loading={isLoading}
+          onCellEditingStopped={onCellEditingStopped}
           onAction={onAction}
+          stopEditingWhenCellsLoseFocus
```

Comment:
`stopEditingWhenCellsLoseFocus` will cause dialog/modal/popout windows to not function properly on the grid eg. date selection cells. 
It would be nice to have this turned on but I wasnt able to find a suitable solution :(

Reply:
gotcha thanks